app/controllers/export/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Export::BaseController < PublicFacingController
+  skip_before_action :set_cache_control_headers
+  skip_before_action :restrict_request_formats
+  before_action :require_export_data_permission!
+  before_action :set_api_cache_control_headers
+  before_action :set_api_access_control_allow_origin_headers
```

Comment:
I don't think we'll need this since we're not requesting this client side, can probably also skip the cache control headers as I doubt we'll be using them.

Reply:
Ok, I'll remove these